tbls/v2/tbls.go

```diff
@@ -59,6 +59,14 @@ type Implementation interface {
 	// Sign signs data with the provided private key, and returns the resulting signature.
 	// This function works on both shares of private keys, and complete private keys.
 	Sign(privateKey PrivateKey, data []byte) (Signature, error)
+
+	// FastAggregateVerify is the BLS standard FastAggregateVerify call, as defined by the standard:
+	// https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-03#section-3.3.4.
+	FastAggregateVerify(shares []PublicKey, signature Signature, data []byte) error
+
+	// CombineSignatures combines signs in a single Signature with standard BLS signature aggregation,
+	// as defined by the standard: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-03#section-2.8.
+	CombineSignatures(signs []Signature) (Signature, error)
```

Comment:
I would rename this to `Aggregate` to align with `ThresholdAggregate` above.

- `ThresholdAggregate` threshold aggregates M-of-N partial signatures.
- `Aggregate` aggregates multiple signatures.